src/main/java/com/diamondfire/helpbot/sys/tag/Tag.java

```diff
@@ -71,14 +71,14 @@ public JsonObject asJson() {
         return json;
     }
     
-    public void sendResponse(TextChannel channel) {
+    public void sendResponse(TextChannel channel, User requester) {
         User user = channel.getJDA().retrieveUserById(getAuthorId()).complete();
         
         EmbedBuilder embed = new EmbedBuilder()
-                .setTitle(getTitle())
+                .setTitle(getTitle()+" (*"+user.getAsTag()+"*)")
                 .setDescription(getResponse()+"\n\u200b")
                 .setColor(0x969dba)
-                .setFooter("Written by "+user.getAsTag(), user.getAvatarUrl());
+                .setFooter("Executed by "+requester.getAsTag(), user.getAvatarUrl());
```

Comment:
Was this change discussed with RyanLand? I just wanna make sure he's cool with this too

Reply:
Yep, we went through a few revisions.